### DISCUSSION TITLE
Proposal|DCDAO Education Writer Program

### DISCUSSION THREAD

**Discussion by Cindy884** [ringdao-community-guild/discussion#25] [Proposals Temperature Check]
# Background

As many members pointed out, sometimes it is difficult to understand the technical knowledge. In addition, DCDAO aims to narrow the gap between the information gap between Darwinia and the community.

That’s why we created the DCDAO Education Writer Program. Through an article competition, participants can know more about web3 and Darwinia.
# ****What will you gain?****

1. You could win a free token. Here are the ranking rewards.
<img width="361" alt="截屏2023-07-03 18 06 48" src="https://github.com/dcdao/discussion/assets/109792098/aee3570a-0db9-484f-a788-4ba2082126fa">

2. The winner will be featured on multiple communications platforms(including the DCDAO Twitter account and Darwinia's official social media account). This is a great résumé-builder for a career in web3 or any relevant field of study.
3. This is an opportunity to become a leading voice in the growing web3 industry and Darwinia community.

# How to participate in it?
1. Click the dework link and follow the [DCDAO](https://app.dework.xyz/darwinia-dao) channel.
<img width="793" alt="截屏2023-07-03 18 00 31" src="https://github.com/dcdao/discussion/assets/109792098/d43f506a-20f2-4807-b3e6-dda1afeb84e6">

2. Read [our event description](https://app.dework.xyz/darwinia-dao/main-workspace-18?taskId=aecf1752-8c56-4090-a3a6-1582575870de) and write a medium article about it. 

4. Click the “you’re interested” button and choose the date you can finish the article.
5. Put your medium post in the message blank.

# ****Awards and selection process****

We will select the top 3 winners to send the rewards based on the clapped numbers and the interactive rate. 
<img width="492" alt="截屏2023-07-03 18 11 07" src="https://github.com/dcdao/discussion/assets/109792098/84c56520-cbd4-4ff2-9a8f-3d289c324b29">

### Notice
1. Event Time: 12/7 00:00 -26/7 23:59 (UTC+8)
2. The article must be written in English.
4. Anyone can apply for it.
5. The prize is listed and paid in RING.
7. The rewards will be issued on dework, so please connect your Metamask wallet when you log in dework. 
8. The rewards will be issued within 7 business days after the event ended. 
9. The decision of DCDAO in respect of any dispute arising out of this program shall be final.

**Comment by SasoLithops** [comment]
I like this idea. We have community members who are good in this(Maduksjay hope you apply) I would raise the reward though to get more interest. By 2 or 3 times.

**Comment by Laki389** [comment]
I like this idea, maybe kanbeok(mariana) would like to participate in this article competition, I will let her know.

**Comment by samestshares** [comment]
This is a welcome idea and it will help the community to understand every piece of information or feature attached to web3 in relation to Darwinia.
